README.md

```diff
@@ -1,5 +1,72 @@
 # Roots Example Project
 
-This repository shows how to use [Bedrock](https://github.com/roots/bedrock) + [bedrock-ansible](https://github.com/roots/bedrock-ansible) + [Sage](https://github.com/roots/sage) (with [Soil](https://github.com/roots/soil)).
+This repository is an example of how to integrate and use the following projects together:
 
-It's deployed on a [Digital Ocean](https://roots.io/r/digitalocean/) 512MB droplet.
+* [Bedrock](https://github.com/roots/bedrock)
+* [bedrock-ansible](https://github.com/roots/bedrock-ansible)
+* [Sage](https://github.com/roots/sage) (with [Soil](https://github.com/roots/soil))
+
+For more background, see this [blog post](https://roots.io/a-modern-wordpress-example/).
+
+This project is a complete working example that's deployed on a [Digital Ocean](https://roots.io/r/digitalocean/) 512MB droplet.
+
+You can view it at http://roots-example-project.com/.
+
+## Instructions
+
+This project can be cloned and re-configured to fit your needs but we highly suggest you follow in the instructions below to create your own. This will guarantee you have the latest version of Bedrock, bedrock-ansible, Sage, and Soil in case this example falls behind a little.
+
+Here's how this example project was created:
+
+1. Create a new project directory: `$ mkdir example.com`
+2. Clone bedrock-ansible: `$ git clone --depth=1 git@github.com:roots/bedrock-ansible.git ansible`
+3. Move `Vagrantfile` to root: `$ mv ansible/Vagrantfile .`
+4. Clone bedrock: `$ git clone --depth=1 git@github.com:roots/bedrock.git site`
+5. Clone Sage: `$ cd site/web/app/themes && git clone --depth=1 git@github.com:roots/sage.git sage`
+6. Delete `.git` folders: `$ rm -rf ansible/.git && rm -rf site/.git && rm -rf site/web/app/themes/sage/.git`
```

Comment:
There is disagreement between steps 5 and 6. In step 5, you CD into themes folder. In step 6, you're back in project root.